tutorials/kubeadm.md

```diff
@@ -57,7 +57,7 @@ location = "user.private.repo"
 
 Next the user should reload and restart the CRI-O service to load the configurations.
 
-Last step. The user also should add the cgroup driver to the configuration file or pass it as a parameter when starting kubeadm for the master node.  The following are samples of parameters for the configuration file:
+The user should not forget to add the cgroup driver. The following are samples of parameters for the configuration file as documented also at [Configure cgroup driver used by kubelet on control-plane node](https://kubernetes.io/docs/setup/production-environment/tools/kubeadm/install-kubeadm/#configure-cgroup-driver-used-by-kubelet-on-control-plane-node):
```

Comment:
This is not at all like the first review.  Did you mean to move to this?  I preferred the prior review fwiw.

Reply:
I will update it I though that the comments from @haircommander was to remove the whole line. I will revert.

Reply:
Done